lib/core.ts

```diff
@@ -219,7 +220,8 @@ type RequiredInstanceOptions = {
     | "validateSchema"
     | "validateFormats"
     | "int32range"
-    | "unicodeRegExp"]: NonNullable<Options[K]>
+    | "unicodeRegExp"
+    | "useRe2"]: NonNullable<Options[K]>
```

Comment:
FWIW, I think this engine is usually spelled "RE2", not "Re2".

Reply:
useRe2 changed to useRE2